tracer/src/Datadog.Trace/DuckTyping/DuckType.cs

```diff
@@ -987,33 +987,26 @@ private static Delegate GetCreateProxyInstanceDelegate(ModuleBuilder? moduleBuil
 
         private static Delegate CreateStructCopyMethod(ModuleBuilder? moduleBuilder, Type proxyDefinitionType, Type proxyType, Type targetType)
         {
-            ConstructorInfo ctor = proxyType.GetConstructors()[0];
+            var ctor = proxyType.GetConstructors()[0];
```

Comment:
This is one area where I actually liked that we used the explicit types 😉

Reply:
[Reverted](https://github.com/DataDog/dd-trace-dotnet/pull/4582/commits/f1963d973ab90de318aed43cda4efc1bd164c2b5)